pkg/cloudprovider/cloudprovider.go

```diff
@@ -61,7 +61,6 @@ import (
 
 func init() {
 	v1alpha5.NormalizedLabels = lo.Assign(v1alpha5.NormalizedLabels, map[string]string{"topology.ebs.csi.aws.com/zone": v1.LabelTopologyZone})
-	corev1beta1.NormalizedLabels = lo.Assign(corev1beta1.NormalizedLabels, map[string]string{"topology.ebs.csi.aws.com/zone": v1.LabelTopologyZone})
```

Comment:
We don't need to add these into the normalized labels anymore?

Reply:
Whoops, just a rebase miss